analyzer/src/main/kotlin/managers/GoMod.kt

```diff
@@ -76,6 +76,10 @@ class GoMod(
 
     companion object {
         const val DEFAULT_GO_PROXY = "https://proxy.golang.org"
+        fun normalizeModuleVersion(moduleVersion: String): String =
```

Comment:
Missing blank line before this one.

Reply:
What's the reason to put this function inside `GoMod`, i.e. why does it fit better here than in `GoDep`? Can we find a more "neutral" place?

Reply:
And the code comment should now ideally become function docs.

Reply:
Can you recommend a better place?

Reply:
I ended up creating a GoSupport file.

Reply:
If that function would be the only content of a new `GoSupport` file, I'd also be fine with following your rationale to put / keep it in `GoMod` as that's the most modern / current package manager for Go.